test/testdrive/logging.td

```diff
@@ -55,3 +55,142 @@ cannot drop schema mz_catalog because it is required by the database system
 
 ! DROP VIEW mz_peek_durations
 cannot drop item mz_catalog.mz_peek_durations because it is required by the database system
+
+? EXPLAIN TYPED PLAN FOR SELECT operator, worker, count FROM mz_arrangement_sharing
```

Comment:
LGTM. I had asked @philip-stoev about how to check the existence of these columns without requiring to provide the expected result and he had `COUNT(*) >= 0` trick up his sleeve.

```
# Verify that all listed columns are available. COUNT(*) >= 0 allows us to avoid
# providing the exact result to testdrive.
> SELECT COUNT(*) >= 0 FROM (SELECT operator, worker, count FROM mz_arrangement_sharing);
true
```

Reply:
I replaced the test with a lookup of `mz_views` and `mz_columns`. Thanks to @philip-stoev for pointing this out!

Reply:
That seems indeed better!